mod/blog/lib/upgrades.json

```diff
@@ -0,0 +1,14 @@
+[
+	{
+		"title": "Blog upgrade",
+		"description": "Lorem ipsum dolor sit amet",
+		"date": 2015052200,
+		"class": "Blog\\Upgrades\\FancyBlogUpgrade2015061700"
+	},
+	{
+		"title": "Another blog upgrade",
+		"description": "Lorem ipsum dolor sit amet",
+		"date": 2015052201,
+		"class": "Blog\\Upgrades\\Bar"
+	}
+]
```

Comment:
This file is currently not being used. The upgrades are instead registered in start.php using `'register', 'upgrades'` hook. See below.